### PR TITLE
[Feature][LineChart] Add main source, need review

### PR DESCRIPTION
**Pull request by filoscoder**
## Description

* Added the main code of the LineChart component inside dooboolab/packages/Charts.
* Created Chart collection README document open for more chart components.
* Has responsive feature for web platforms
* Tested on web / iOS/ android

## Tests
Basic render test code 

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,9 @@
     "install-packages": "lerna exec -- yarn"
   },
   "dependencies": {
-    "dooboolab-welcome": "^1.2.0"
+    "d3": "^5.16.0",
+    "dooboolab-welcome": "^1.2.0",
+    "react-native-svg": "^12.1.0"
```
Comment (hyochan) [inline package.json]: Please remove the dependencies in the root! Only `dooboolab-welcome` is permitted here.

**Review by hyochan — CHANGES_REQUESTED** [review]
Please move your packages to `packages/` instead of `main/`. Your library would rely on separate packages as `@dooboo-ui/charts`. More information in #194 

There will be `BarChart`, `LineChart`, and `PieChart` in `@dooboo-ui/charts`. Also, don't forget to rebase this.

**Comment by filoscoder** [comment]
Fixed requested changes and re-requested review. Please check the changes and let me know if it needs more updates.

TODO:
* Test-code not written yet

**Comment by filoscoder** [comment]
Requesting review before merge! @hyochan

File: packages/Charts/LineChart/README.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+  ```
+
+- Data
+
+  ```javascript
+  state = {
+    data: [
+      {
+        id: 'abcd1234efgh5674',
+        key1: '6일차',
```
Comment (hyochan) [inline packages/Charts/LineChart/README.md:117]: no Korean please!

File: packages/Charts/LineChart/types.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  withLine?: boolean; // default: true
+  lineColor?: string; // default: ('#000000')
+  lineStrokeWidth?: number; // 1
+  withText?: boolean; // default: true
+  textColor?: string; // default: ('#000000')
+  textStrokeColor?: string; // default: ('none')
+  fontSize?: string; // default: ('12')
+  fontWeight?: string; // default: ('bold')
+};
+type YStyle = {
```
Comment (hyochan) [inline packages/Charts/LineChart/types.ts:33]: Adding new line here would be clearer

File: packages/__tests__/LineChart.test.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+import { LineChart } from '../Charts';
+
+let props: any;
+let component: React.ReactElement;
+let testingLib: RenderResult;
+
+const testingData = [
+  {
+    id: 'abcd1234efgh5674',
+    key1: '6일차',
```
Comment (hyochan) [inline packages/__tests__/LineChart.test.tsx:17]: No Korean in tests either!

File: stories/packages/LineChart.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+  width: 100%;
+  padding: 10px;
+  background-color: rgba(245, 245, 245, 1);
+  text-align: center;
+`;
+
+const mockData = [
+  {
+    id: 'abcd1234efgh5674',
+    key1: '6일차',
```
Comment (hyochan) [inline stories/packages/LineChart.stories.tsx:25]: No Korean.

**Review by hyochan — CHANGES_REQUESTED** [review]
Looks good to be merged after fixing minor reviews! 👍

File: packages/Charts/LineChart/README.md [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+lineWidth: 2,
+withDots: true,
+dotColor: '#ffffff',
+dotStrokeColor: '#000000',
+dotStrokeWidth: 2,
+withText: true,
+textColor: '#000000',
+textStrokeColor: 'none',
+fontSize: '12',
+fontWeight: 'bold'
```
Comment (hyochan) [inline packages/Charts/LineChart/README.md:58]: Indentation?

File: packages/Charts/LineChart/README.md [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+          withIndicator: true,
+          withLine: true,
+          lineColor: '#000000', // hex | rgba | string: 'red' | 'black'
+          lineStrokeWidth: 1,
+          withText: true,
+          textColor: '#000000', // hex | rgba | string: 'red' | 'black'
+          textStrokeColor: 'none', // hex | rgba | string: 'red' | 'black'
+          fontSize: '12',
+          fontWeight: 'bold',
+}
```
Comment (hyochan) [inline packages/Charts/LineChart/README.md:93]: Indentation?

File: packages/Charts/LineChart/README.md (hunk 2 above)
Comment (filoscoder) [reply packages/Charts/LineChart/README.md:117]: got it!

File: packages/Charts/LineChart/types.ts (hunk 3 above)
Comment (filoscoder) [reply packages/Charts/LineChart/types.ts:33]: You're right, got it!

**Review by hyochan — APPROVED** [review]
Next time please continue to commit to the same branches when reviewed.
Closed via #233

**Pull request merged**